rootfs/api/models/__init__.py

```diff
@@ -142,6 +143,12 @@ def _log_config_updated(**kwargs):
     config.app.log("config {} updated".format(config))
 
 
+def _log_app_settings_updated(**kwargs):
+    appSettings = kwargs['instance']
+    # log only to the controller; this event will be logged in the release summary
+    appSettings.app.log("application settings {} updated".format(appSettings))
```

Comment:
This just says that settings have been updated but not what all has changed. app.log will call logger.log as with level as info
If Debug mode is on then user will see the full details of what all has changed which is what i had done above.